pkg/cluster/cluster_info_scraper.go

```diff
@@ -66,17 +69,21 @@ type ClusterScraper struct {
 	DynamicClient           dynamic.Interface
 	ControllerRuntimeClient runtimeclient.Client
 	cache                   turbostore.ITurboCache
+	GitOpsConfigCache       *map[string][]*gitopsv1alpha1.Configuration
```

Comment:
maps in go are already pointers and can be exactly used as pointers. You would almost never need a pointer to a map.

Reply:
Removed the pointer.